_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/includes/developer-materials-results.html.twig

```diff
@@ -12,77 +12,81 @@
                     </div>
-                    <p>{{ '{{' }} item.fields.sys_description | truncate {{ '}}' }}&hellip;</p></div>
+                    <p ng-bind-template="{{ '{{' }} item.fields.sys_description | truncate {{ '}}' }}&hellip;"></p>
+                </div>
             </li>
         </ul>
     </div>
-    <div class="panel" ng-hide="data.materials.length || data.loading">Sorry there were no results, please modify your
-        search query.
-    </div>
+    <div class="panel" ng-hide="data.materials.length || data.loading" ng-bind-template="Sorry there were no results, please modify your search query."></div>
 </div>
 <div class="large-24" ng-hide="data.loading">
-    <div class="large-7 columns" id="show-entries"><label class="inline">Show</label><select name="show-entries"
-                                                                                             ng-change="fetchAndUpdate();"
-                                                                                             ng-model="pagination.size">
+    <div class="large-7 columns" id="show-entries">
+        <label class="inline">Show</label>
+        <select name="show-entries" ng-change="fetchAndUpdate();" ng-model="pagination.size">
             <option value="10">10</option>
             <option value="15">15</option>
             <option value="20">20</option>
             <option value="25">25</option>
-        </select><label class="inline">entries</label><br/>
-        <p>
-            Showing {{ '{{' }} (paginate.currentPage * pagination.size - pagination.size + 1) | safeNumber {{ '}}' }}
-            to {{ '{{' }} Math.min(data.total, paginate.currentPage * pagination.size) | safeNumber {{ '}}' }}
-            of {{ '{{' }} data.total | safeNumber {{ '}}' }} entries</p></div>
+        </select>
+        <label class="inline">entries</label>
+        <br/>
+        <p ng-bind-template="Showing {{(paginate.currentPage * pagination.size - pagination.size + 1) | safeNumber}} to {{ Math.min(data.total, paginate.currentPage * pagination.size) | safeNumber}} of {{data.total | safeNumber}} entries"></p>
+    </div>
     <div class="large-17 small-24 columns">
         <nav id="paginator" ng-hide="pagination.viewall || !data.materials.length || data.loading">
             <ul class="pagination">
-                <li ng-class="{unavailable: paginate.currentPage < 2}"><a ng-click="firstPage()">First</a></li>
-                <li ng-class="{unavailable: paginate.currentPage < 2}"><a ng-click="previousPage()">Previous</a></li>
-                <li ng-class="{current: page == paginate.currentPage}"
-                    ng-repeat="page in paginate.pagesArray track by $index"><a
-                            data-page="{{ '{{' }} page {{ '}}' }}"
-                            ng-click="goToPage(page)">{{ '{{' }} page {{ '}}' }}</a>
+                <li ng-class="{unavailable: paginate.currentPage < 2}">
```

Comment:
@LightGuard I only fixed the indentation on this part, but left everything else the way you had written it which I believe works well. I'll add the second curly braces and test it out